src/index.js

```diff
@@ -285,8 +285,12 @@ class Searcher extends NxusModule {
       factor: 2,
       maxAttempts: this.config.retryAttempts,
       handleError: (err, context) => {
-        this.log.trace("Retry attempt errored", err, Object.keys(err))
-        if (! (err.status && err.status == 429) ) {
+        let status = err.originalError && err.originalError.status
```

Comment:
This should be `statusCode` it appears from @davidkellerman testing not `status`